benchmarks/nel/project.yml

```diff
@@ -4,27 +4,38 @@ vars:
   name: "nel_benchmark"
   config: "nel.cfg"
   vectors_model: "en_core_web_md"
-  version: "0.0.1"
+  version: "0.0.3"
+  dataset: "mewsli_9"
+  gpu_id: ""
+  use_filtered_dumps: "--use_filtered_dumps"
+  training_max_steps: 1000
 
 directories: ["assets", "training", "configs", "scripts", "corpora", "temp"]
 
 assets:
-  - dest: 'assets/reddit.zip'
-    url: 'https://zenodo.org/record/3970806/files/reddit_el.zip?download=1'
-    checksum: 'eea345cc7574a5c9c376748d1871d557'
-    description: Entity linking dataset scraped from Reddit. See [paper](https://arxiv.org/abs/2101.01228).
   - dest: 'assets/wiki/wikidata_entity_dump.json.bz2'
     url: 'https://dumps.wikimedia.org/wikidatawiki/entities/latest-all.json.bz2'
-    description: Wikidata entity dump.
+    description: Wikidata entity dump. Download can take a long time!
+    extra: True
   - dest: 'assets/wiki/wikipedia_dump.xml.bz2'
     url: 'https://dumps.wikimedia.org/enwiki/latest/enwiki-latest-pages-articles-multistream.xml.bz2'
-    description: Wikipedia dump.
+    description: Wikipedia dump. Download can take a long time!
+    extra: True
+  - dest: 'assets/wiki/wikidata_entity_dump_filtered.json.bz2'
+    url: 'https://github.com/explosion/projects/releases/download/nel-benchmark-filtered-wiki-data/wikidata_entity_dump_filtered.json.bz2'
+    description: Filtered Wikidata entity dump for demo purposes.
+    checksum: 'ba2d979105abf174208608b942242fcb'
+  - dest: 'assets/wiki/wikipedia_dump_filtered.xml.bz2'
+    url: 'https://github.com/explosion/projects/releases/download/nel-benchmark-filtered-wiki-data/wikipedia_dump_filtered.xml.bz2'
+    description: Filtered Wikipedia dump for demo purposes.
+    checksum: 'cb624eaa5887fe1ff47a9206c9bdcfd8'
 
 workflows:
   all:
-    - setup
+    - download_mewsli-9
```

Comment:
I always find it awkward to type/remember commands that include both a `_` as well as a `-`. Can we maybe shorten this to `download_mewsli`?

Reply:
Renamed it to `download_mewsli9`. If you prefer `download_mewsli`, feel free to reopen this conversation and I'll adjust it.